content/source/docs/enterprise-beta/api/workspaces.html.md

```diff
@@ -26,9 +26,10 @@ The default `/workspaces` endpoint creates a workspace without configuring the V
 ### Parameters
 
 - `:organization` (`string: <required>`) - Specifies the username or organization name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
+- `o-auth-token-id` (`string: <optional>`) - Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtain from the [o-auth-tokens](/docs/enterprise-beta/api/o-auth-tokens.html) endpoint.
```

Comment:
\*obtained

This should be a relative link (`./o-auth-tokens.html`) to make it more resilient when we eventually relocate it.